src/mscorlib/src/System/Globalization/DateTimeFormat.cs

```diff
@@ -993,7 +1020,17 @@ internal static string FastFormatRoundtrip(DateTime dateTime, TimeSpan offset)
 
             return StringBuilderCache.GetStringAndRelease(result);
         }
-        
+
+        private static void AppendTimeOfDay(StringBuilder result, DateTime dateTime)
```

Comment:
Nit: the name should probably include format information, e.g. AppendHHmmssTimeOfDay, or something along those lines.